### ISSUE TITLE
Native Stripe Connect Account Creation

### ISSUE THREAD

**Issue by sbarbour-93** [stripe/stripe-react-native#842]
**Is your feature request related to a problem? Please describe.**

_I'd like the ability for my customers to create a Stripe Connected account via a native flow. Currently the solution is to open a web view and eventually re-direct back into my app once completed. This can present a few problems if the user leaves the account onboarding flow mid-process._

**Describe the solution you'd like**

_I'd like for the Stripe React Native SDK to expose API methods that would allow the Stripe Connected Account creation flow to be completed natively without the need for the introduction of webviews and deep linking._

**Describe alternatives you've considered**

_Opening a webview within the app that allows my customers to create a Stripe Connected Account via the website._

**Additional context**

**Comment by fdelacruzsoto** [comment]
@charliecruzan-stripe is anyone working on this?

**Comment by charliecruzan-stripe** [comment]
Is this the functionality you're looking for- https://stripe.com/docs/financial-connections/connect-payouts?platform=web? 

If so, it's in progress here- https://github.com/stripe/stripe-react-native/pull/1049

**Comment by petkokp** [comment]
Any updates on this? Has anyone managed to find a workaround?

**Comment by shreypdev** [comment]
Any updates? Can we prioritize this? Lot of people are having same issue

**Comment by AnasMK9** [comment]
same here, this looks like it's related to opening stripe's flow inside a webview, where it gets stuck on the phone verification and recaptcha parts and keeps restarting, same issue seems consistent on both flutter and react native

**Comment by iFabrycio** [comment]
Hello there, any updates on this?

**Comment by MuzooraBarnabas** [comment]
2 years and some weeks without any response. Whats the work around?

**Comment by Umar-Farooq-Shafi** [comment]
any update?

**Comment by cttsai-stripe** [comment]
We now have a mobile SDK that supports onboarding Stripe Connected Accounts from your mobile app. It still uses a webview but it should be less flaky than implementing it on your own. Would this work for your use case?
- [iOS Get started](https://docs.stripe.com/connect/get-started-connect-embedded-components?platform=ios) / [iOS Account onboarding embedded component](https://docs.stripe.com/connect/supported-embedded-components/account-onboarding?platform=ios)
- [Android Get started](https://docs.stripe.com/connect/get-started-connect-embedded-components?platform=android) / [Android Account onboarding embedded component](https://docs.stripe.com/connect/supported-embedded-components/account-onboarding?platform=android)

**Comment by whybtech** [comment]
Just to clarify, the onboarding embedded components mentioned are designed for native iOS and Android, not for React Native.
This issue should remain open because we still need support for using an Account Token to onboard connected accounts (see #427).

This is the first step required to [create a connected account](https://docs.stripe.com/api/accounts/create#create_account-account_token), where you must pass an [account token](https://docs.stripe.com/api/tokens/create_account).
If a webview is needed afterward, you can then [create an account link](https://docs.stripe.com/api/account_links/create).

**Comment by AntoninMarchardDev** [comment]
Agreed wuth @whybtech , this issue need to remains open. The only secure solution is webview, and it's not the perfect one. Since we cannot generate account token server side because of Stripe, we need at least a method createAccountToken exposed from stripe react native.

**Comment by corentinclichy** [comment]
Hello, 

I really need this feature too, is there any news on that ? 

thanks for the work !
